controllers/pvc_storageclass_test.go

```diff
@@ -138,20 +139,23 @@ var _ = Describe("The default StorageClass has not been configured", func() {
 				Expect(k8sClient.Delete(ctx, &pvc)).Should(Succeed())
 			}()
 
-			By("Expecting the PVC has to be unchanged")
-			Consistently(func() corev1.PersistentVolumeClaim {
+			By("Expecting the PVC hasn't annotated with StorageClass")
```

Comment:
sorry, just a nit but it's bugging me:
`Expecting the PVC hasn't annotated with StorageClass` -> `Expecting the PVC not to have the StorageClass annotation`.